.github/workflows/maven.yaml

```diff
@@ -33,27 +41,35 @@ jobs:
 
       - name: build controllers project
         uses: ./.github/workflows/composites/build-controllers-project
+        if: env.BASE_BRANCH == 'main'
```

Comment:
skip a few steps that are strictly related to integration tests, that we do not run on `2.1.x`